salt/master/pillar.sls

```diff
@@ -37,7 +37,9 @@ include:
 - reclass.storage.data
 
 /srv/salt/reclass/classes/service:
-  file.directory
+  file.directory:
+  - require:
+    - reclass_data_dir
```

Comment:
This looks like wrong syntax, no module specified, and should it depend on git resource?